app/services/preserved_object_handler.rb

```diff
@@ -219,8 +222,7 @@ def update_online_version(moab_validated=false, status=nil)
       pres_object = PreservedObject.find_by!(druid: druid)
       pres_copy = PreservedCopy.find_by!(preserved_object: pres_object, endpoint: endpoint) if pres_object
 
-      # TODO: add this later with audit timestamp updates
-      # raise_rollback_if_pc_po_version_mismatch(pres_copy.version, pres_object.current_version)
+      raise_rollback_if_pc_po_version_mismatch(pres_copy.version, pres_object.current_version)
```

Comment:
@jmartin-sul : I think it was because this PR was done in tandem with some refactoring in a separate PR, and I commented out the line in one context, waiting for the other.  Something like that, anyway.